test/Geo_Tests/src/Geo_Spec.enso

```diff
@@ -4,6 +4,7 @@ from Standard.Table.Data.Table.Table import Table_Data
 import Standard.Geo
 
 from Standard.Test import Test
+import Standard.Test.Main as Test_Module
```

Comment:
Think we can use `import Standard.Test.Extensions` for these.

Reply:
done